Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -70,16 +46,15 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
         }
     }
     
-    init(name: String, shortDescription: String = "") {
-        self.name = name
-        self.customShortDescription = shortDescription
+    init(options: ProcessingOptions) {
+        self.options = options
     }
     
     // MARK: - Command implementation
     
     func execute() throws {
-        guard let inputPath = inputPath else { exit(1) }
-        guard let configPath = configPath else { exit(1) }
+        guard let inputPath = options.inputPath else { exit(1) }
+        guard let configPath = options.configPath else { exit(1) }
```

Comment:
Check [Validation](https://github.com/apple/swift-argument-parser/blob/main/Sources/ArgumentParser/Documentation.docc/Articles/Validation.md) guide, maybe it's better to use it to make errors more descriptive. But also, we can simply require these path arguments.